nncf/experimental/post_training/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -0,0 +1,334 @@
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.graph.transformations.layout import TransformationLayout
+from nncf.common.utils.backend import BackendType, get_backend
+from nncf.common.graph.transformations.commands import TargetPoint
+from nncf.experimental.post_training.algorithms import AlgorithmParameters
+from nncf.experimental.post_training.algorithms.algorithm import Algorithm
+from nncf.experimental.post_training.algorithms.algorithm import PostTrainingAlgorithms
+from nncf.experimental.post_training.algorithms.fast_bias_correction.backend import ALGO_BACKENDS
+from nncf.experimental.post_training.api.engine import Engine
+from nncf.experimental.post_training.factories import NNCFGraphFactory
+from nncf.experimental.post_training.graph.model_transformer import StaticModelTransformerBase
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPoint
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPointsContainer
+
+ModelType = TypeVar('ModelType')
+
+
+class FastBiasCorrectionParameters(AlgorithmParameters):
+    """
+    Parameters of FastBiasCorrection algorithm
+
+    Args:
+        number_samples:
+            The number of the samples for the statistics collection.
+            This statistics uses for the further calculation of the bias shift.
+        threshold:
+            The magnitude threshold that regulates the application of the shift.
+            Magnitude calculates as the maximum of the absolute ratio of the shift to the original bias value.
+            If the calculated value less than threshold, shift will apply to the bias.
+
+    """
+
+    def __init__(self, number_samples: int = 100, threshold: float = 2.0) -> None:
+        self.number_samples = number_samples
+        self.threshold = threshold
+
+    def to_json(self) -> Dict[str, Union[str, float, int]]:
+        """
+        Serialize all FastBiasCorrection parameters to JSON.
+        """
+
+
+class FastBiasCorrection(Algorithm):
+
+    """
+    Post-training FastBiasCorrection algorithm implementation
+
+    The main purpose of this algorithm to reduce quantization error
+    via correction the bias of the Convolutions, FullyConnected, etc. layers.
+    The algorithm pipeline is very simple:
+        - we collects floating-point statistics from the corresponding model for the layers with bias;
+        - then we gets the quantized model and try to reduce it's error by correction of the bias;
+        - the shift calculates using the sub-graph that consists of the correction layer and
+        weight quantizer-dequantizer pair or fake quantize node;
+        - the floating-point statistics uses as input for
+        the sub-graph and further quantization output calculation;
+        - in the end we corrects the original bias by the difference (shift)
+        between floating-point and quantized outputs;
+
+    Args:
```

Comment:
Please use format from the https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#352-functions-and-methods

Reply:
Done. Please, review it again.